docs/compute-vm-quickstart.md

```diff
@@ -0,0 +1,136 @@
+# Quickstart for Using the Oracle Toolkit for Google Cloud on Compute Engine VMs
+
+This document serves as a quickstart guide and simple reference for using the Oracle Toolkit for Google Cloud on Compute Engine Virtual Machines. It is an abridged version of the more comprehensive [Oracle Toolkit for Google Cloud - Compute Engine VM User Guide](compute-vm-user-guide.md).
+
+## Prerequisite Assumptions
+
+This document assumes that you have:
+
+1. Provisioned your Ansible Control Node, installed Ansible and JMESpath on it, and downloaded the toolkit.
+1. Setup Google Cloud foundational components such as IAM, networking, Google Cloud Storage buckets with the required media staged, and security aspects.
```

Comment:
In the spirit of a quickstart, what needs to be set up to get up and running quickly?  Again, I'm not sure there are too many steps here:
- Create a VPC
- Create a GCS bucket for Oracle software
- Run `check-swlib.sh` to figure out what files you need
- Go to https://support.oracle.com, and search and download each file name  (Or https://www.oracle.com/downloads/#category-database for base releases)
- gcloud storage cp *.zip gs://my-bucket